app/config/locale/translations/ko.json

```
@@ -3,30 +3,30 @@
     "settings.locale": "ko",
     "settings.direction": "ltr",
     "emails.sender": "%s 팀",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "계정 인증",
+    "emails.verification.hello": "안녕하세요 {{name}}님",
+    "emails.verification.body": "이메일 인증을 위해 링크를 클릭하여주세요.",
+    "emails.verification.footer": "이메일 인증을 부탁하지 않으셨다면 이 메시지를 무시하여주세요.",
+    "emails.verification.thanks": "감사합니다",
+    "emails.verification.signature": "{{project}} 팀",
+    "emails.magicSession.subject": "로그인",
+    "emails.magicSession.hello": "안녕하세요,",
+    "emails.magicSession.body": "로그인 하시려면 링크를 클릭하여주세요.",
+    "emails.magicSession.footer": "이 이메일 계정으로 로그인 신청을 하지 않으셨다면 이 메세지를 무시하여주세요.",
+    "emails.magicSession.thanks": "감사합니다",
+    "emails.magicSession.signature": "{{project}} 팀",
+    "emails.recovery.subject": "비밀번호 재설정",
+    "emails.recovery.hello": "안녕하세요 {{name}}님",
+    "emails.recovery.body": "{{project}}의 비밀번호 재설정을 위해 링크를 클릭하여주세요.",
+    "emails.recovery.footer": "비밀번호 재설정 신청을 하지 않으셨다면 이 메세지를 무시하여주세요.",
+    "emails.recovery.thanks": "감사합니다",
+    "emails.recovery.signature": "{{project}} 팀",
+    "emails.invitation.subject": "초대장 %s 팀 - %s",
+    "emails.invitation.hello": "안녕하세요",
```

```suggestion
    "emails.invitation.hello": "안녕하세요,",
```
Can we make sure punctuation is taken care of in all the strings ?
https://github.com/appwrite/appwrite/blob/master/app/config/locale/translations/en.json#L25